compact.go

```diff
@@ -440,7 +442,7 @@ func (c *LeveledCompactor) Write(dest string, b BlockReader, mint, maxt int64, p
 		return ulid.ULID{}, nil
 	}
 
-	level.Info(c.logger).Log("msg", "write block", "mint", meta.MinTime, "maxt", meta.MaxTime, "ulid", meta.ULID)
+	level.Info(c.logger).Log("msg", "write block", "mint", meta.MinTime, "maxt", meta.MaxTime, "ulid", meta.ULID, "duration", time.Since(start))
```

Comment:
Nit. To keep the line width sane, can you break each log entity into a single line like this?
https://github.com/prometheus/tsdb/blob/dac2b97dfdf354a874f186f365fa75cd647e755e/compact.go#L391-L399

Reply:
+1 - like in my example in the comment https://github.com/prometheus/tsdb/pull/505#issuecomment-456111128

Reply:
Thx, formatting updated